README.md

```diff
@@ -25,6 +25,22 @@ For instance, use the following command to use the `master` branch of Symfony:
 STABILITY=dev docker-compose up --build
 ```
 
+## Build and running behind a proxy
+
+1. Uncomment in the Dockerfile the following line
+```bash
+RUN pear config-set http_proxy yourproxy
+```
+2. Replace "yourproxy" by your proxy
+3. Build
+```bash
+docker-compose build --build-arg http_proxy="yourproxy" --build-arg https_proxy="yourproxy"
```

Comment:
I have never used pear through a proxy and I don't know this part very well but can we expose http_proxy & https_proxy in the Dockerfile and use the ENV variable like this : `PROXY="http://yourproxy.com/" docker-composer build` ?
This would delete step 2

Reply:
@maxhelias for the docker-compose I'm not sure you can build this way, isn't env var but a build-arg.
If I understand using PROXY="" before docker-compose will set a ENV proxy to docker-compose which can be used when building a Dockerfile but I don't know if it can be used this way with docker-compose.

Anyway, we can use the env and inject it into the Dockerfile when we use pear.

Reply:
For `PROXY`, i mean like `SYMFONY_VERSION` https://github.com/dunglas/symfony-docker/blob/d5cc26520a156731110e5c3cf486d2aa056ff5e9/Dockerfile#L76

https://github.com/dunglas/symfony-docker/blob/d5cc26520a156731110e5c3cf486d2aa056ff5e9/Dockerfile#L79